### PR TITLE
Add Apply for Div page to Solicitor Create

### PR DESCRIPTION
**Pull request by tomxelliott**

### CODE REVIEW

**Review by nitinprabhu — APPROVED** [review]
I assume this is locally tested ?

**Comment by nitinprabhu** [comment]
> **Before creating a pull request make sure that:**
> 
> * [ ]  commit messages are meaningful and follow good commit message guidelines
> * [ ]  README and other documentation has been updated / added (if needed)
> * [ ]  tests have been updated / new tests has been added (if needed)
> 
> Please remove this line and everything above and fill the following sections:
> 
> ### JIRA link (if applicable)
> ### Change description
> **Does this PR introduce a breaking change?** (check one with "x")
> 
> ```
> [ ] Yes
> [ ] No
> ```



> **Before creating a pull request make sure that:**
> 
> * [ ]  commit messages are meaningful and follow good commit message guidelines
> * [ ]  README and other documentation has been updated / added (if needed)
> * [ ]  tests have been updated / new tests has been added (if needed)
> 
> Please remove this line and everything above and fill the following sections:
> 
> ### JIRA link (if applicable)
> ### Change description
> **Does this PR introduce a breaking change?** (check one with "x")
> 
> ```
> [ ] Yes
> [ ] No
> ```

we normally remove the default template and add details like what functionality is added by PR and jira id

**Comment by tomxelliott** [comment]
@nitinprabhu Not been able to test everything locally yet as I don't have full access. Also with regards to Jira number, Linus said there isn't a ticket for this - also I don't yet have access to Jira.

File: src/main/java/uk/gov/hmcts/divorce/solicitor/event/page/JurisdictionApplyForDivorce.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                "labelResidualJurisdictionBullet-1",
+                "• The petitioner and the respondent married each other in England and Wales; and"
+            )
+            .label(
+                "labelResidualJurisdictionBullet-2",
+                "• Neither the petitioner nor the respondent is able to apple for a divorce in any other country; and"
+            )
+            .label(
+                "labelResidualJurisdictionBullet-3",
+                "• It would be in the interests of justice for hte court to consider the application "
```
Comment (linusnorton) [inline src/main/java/uk/gov/hmcts/divorce/solicitor/event/page/JurisdictionApplyForDivorce.java:72]: `hte` typo I assume

Comment (tomxelliott) [reply src/main/java/uk/gov/hmcts/divorce/solicitor/event/page/JurisdictionApplyForDivorce.java:72]: Well spotted!

**Review by linusnorton — APPROVED** [review]

**Pull request merged**